AppAuth-JS/app/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
```

Comment:
add license header

Reply:
this is Google's lib fork. Seems Google has missed license, added WSO2 license since I edited it